dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/Relationship.java

```diff
@@ -43,6 +43,8 @@
 @AllArgsConstructor
 public class Relationship implements TrackerDto
 {
+    private String uid;
```

Comment:
I am removing the String uid fields from the other models, since we already have fields for uid (trackedEntity, enrollment, event, relationship).

I would like you to use relationship instead here, since the only change we would potentially do here, is to add a new uid if there is no uid already.

Reply:
the "relationship" field on Relationship I mean :)

Reply:
resolved